ironfish-cli/package.json

```diff
@@ -45,6 +45,7 @@
     "start:dev": "node start",
     "start": "yarn build && yarn start:js",
     "start:js": "cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node --expose-gc bin/run",
+    "start:debug": "yarn build && cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node --inspect bin/run",
```

Comment:
Do you want the GC exposed on this too? I'm not sure if it's automatically exposed as part of --inspect

Reply:
I suspect it isn't, but might come in handy for debugging! Added